js/clock.js

```diff
@@ -0,0 +1,100 @@
+/*Craig Bergmeyer
+clock.js
+INFO 2134
+Thoendel
+May 3, 2020*/
+
+//from https://www.w3schools.com/graphics/canvas_clock.asp
```

Comment:
Nice and clean here. Really appreciate how you format things for maintenance and readability.